families/settings/tests/test_settings.py

```diff
@@ -0,0 +1,118 @@
+import os
+import shutil
+import tempfile
+from collections import OrderedDict
+from sawtooth_settings.processor.config.settings \
+    import merge_settings_config
+from sawtooth_settings.processor.config.settings \
+    import load_toml_settings_config
+from sawtooth_settings.processor.config.settings \
+    import load_default_settings_config
+from sawtooth_sdk.processor.exceptions import LocalConfigurationError
+from sawtooth_processor_test.transaction_processor_test_case \
+    import TransactionProcessorTestCase
+
+
+class TestSettings(TransactionProcessorTestCase):
+
+    def test_load_toml_settings_config(self):
+        """ the test case will create the toml file at a per the environment
+         and the path set,open the file ,write the content in the file and the
+        instantiate the object through this file by the help of the function
+        """
+        orig_environ = dict(os.environ)
+        os.environ.clear()
+        directory = tempfile.mkdtemp(prefix="test-path-config-")
+
+        try:
+            os.environ['SAWTOOTH_HOME'] = directory
+            config_dir = os.path.join(directory, 'etc')
+            os.mkdir(config_dir)
+            filename = os.path.join(config_dir, 'families_settings.toml')
+            with open(filename, 'w') as fd:
+                fd.write('connect= "tcp://localhost:4004"')
+                fd.write(os.linesep)
+
+            config = load_toml_settings_config(filename)
+            self.assertEqual(config.connect, "tcp://localhost:4004")
+
+        finally:
+            os.environ.clear()
+            os.environ.update(orig_environ)
+            shutil.rmtree(directory)
+
+    def test_load_default_settings_config(self):
+        """Tests the default configuration.
+            connect = "tcp://localhost:4004" """
+        settings = load_default_settings_config()

```

Comment:
The ending """ needs to be on a new line. The second line needs to be aligned properly.